### PR TITLE
Metrics-output-improvements

### PR DESCRIPTION
**Pull request by ruheni**
Wrap metrics output in a CodeWithResult component

### CODE REVIEW

**Review by janpio — CHANGES_REQUESTED** [review]
See Slack comment

File: content/200-concepts/100-components/02-prisma-client/240-metrics.mdx [hunk 1]
```diff
@@ -89,6 +89,36 @@ console.log(metrics)
 This returns metrics as follows:
 
 ```json
+{
+  "counters": [
+    {
+      "key": "query_total_operations",
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/240-metrics.mdx]: This does not look like a Prisma metric.

File: content/200-concepts/100-components/02-prisma-client/240-metrics.mdx [hunk 2]
```diff
@@ -89,6 +89,36 @@ console.log(metrics)
...
+      "labels": { "server": "us_server1", "app_version": "one" },
+      "value": 0,
+      "description": "The total number of operations executed"
+    },
+    {
+      "key": "prisma_datasource_queries_total",
+      "labels": { "server": "us_server1", "app_version": "one" },
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/240-metrics.mdx]: labels should be empty here or not even present for a plain `prisma.$metrics.json()`

**Review by janpio — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/240-metrics.mdx (hunk 2 above)
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/240-metrics.mdx]:
```suggestion
      "labels": { },
```

File: content/200-concepts/100-components/02-prisma-client/240-metrics.mdx [hunk 3]
```diff
@@ -89,6 +89,36 @@ console.log(metrics)
...
+      "value": 0,
+      "description": "The total number of queries executed"
+    },
```
Comment (ruheni) [inline content/200-concepts/100-components/02-prisma-client/240-metrics.mdx]:
```suggestion
    {
      "key": "prisma_client_queries_total",
      "labels": {},
      "value": 2,
      "description": "Total number of Prisma Client queries executed"
    },
    {
      "key": "prisma_datasource_queries_total",
      "labels": {},
      "value": 5,
      "description": "Total number of Datasource Queries executed"
    },
    {
      "key": "prisma_pool_connections_open",
      "labels": {},
      "value": 1,
      "description": "Number of currently open Pool Connections"
    }
```

**Pull request merged**